incubator/hnc/internal/pkg/selectors/selectors.go

```diff
@@ -33,6 +39,9 @@ func GetTreeSelector(inst *unstructured.Unstructured) (labels.Selector, error) {
 	strs := strings.Split(treeSelectorStr, ",")
 	selectorStr := ""
 	for i, str := range strs {
+		if str == "" {
+			continue
+		}
```

Comment:
Are you going to remember why this is here in six months? :)

Reply:
Added comment here